### ISSUE TITLE
call owner_api  occur can't create Tokio runtime

### ISSUE THREAD

**Issue by OnfireMrHuang** [mimblewimble/grin-wallet#494]
exec /v3/owner, method is retrieve_txs,I got this:

time="2020-07-26T11:15:04+08:00" level=error msg="OwnerAPI: RPC Error during OpenWallet" code=-32099 message="Lifecycle: Error opening wallet: Wallet store error: LMDB error: Too many open files , Data Dir: /home/ubuntu/.grin/main/wallet_data"

and owner server print error message:

200726 17:08:01.438 ERROR grin_wallet_impls::node_clients::http - Error calling get_tip: Internal error: can't create Tokio runtime
20200726 17:08:01.438 WARN grin_wallet_libwallet::api_impl::owner_updater - Updater Thread unable to contact node
20200726 17:08:01.584 ERROR grin_wallet_impls::node_clients::http - Error calling get_tip: Internal error: can't create Tokio runtime
20200726 17:08:01.585 WARN grin_wallet_libwallet::api_impl::owner_updater - Updater Thread unable to contact node
20200726 17:08:01.731 ERROR grin_wallet_impls::node_clients::http - Error calling get_tip: Internal error: can't create Tokio runtime
20200726 17:08:01.733 WARN grin_wallet_libwallet::api_impl::owner_updater - Updater Thread unable to contact node
20200726 17:09:02.812 ERROR grin_wallet_impls::node_clients::http - Error calling get_tip: Internal error: can't create Tokio runtime
20200726 17:09:02.812 WARN grin_wallet_libwallet::api_impl::owner_updater - Updater Thread unable to contact node

**Comment by yeastplume** [comment]
Does this happen consistently? How long has the wallet been running? It looks like it may be a ulimit issue

**Comment by chainlife-doc** [comment]
> Does this happen consistently? How long has the wallet been running? It looks like it may be a ulimit issue

yes, I solves this problem by set the open file descriptor to 65536

**Comment by ardocrat** [comment]
Seems not actual
